File: 06-Problem-Variables/06-Problem-Variables.swift
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

var myMutable = 4;
let MY_IMMUTABLE = 5;

print("myMutable: \(myMutable) myMutable MY_IMMUTABLE: \(MY_IMMUTABLE)");

myMutable += MY_IMMUTABLE;

print("myMutable: \(myMutable) myMutable MY_IMMUTABLE: \(MY_IMMUTABLE)");

File: 06-Problem-Variables/06_Problem_Variables.py
# Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

myMutable = 4
MY_IMMUTABLE = 5

print(f"myMutable: {myMutable} MY_IMMUTABLE: {MY_IMMUTABLE}")

myMutable += MY_IMMUTABLE

print(f"myMutable: {myMutable} MY_IMMUTABLE: {MY_IMMUTABLE}")
